spec/integration/events_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe "Events" do
+  let(:application) { create(:application) }
+  let(:raw_event)   { create(:raw_event) }
+
+  before do
+    worker = ParseRawEventJob.new
+    worker.perform raw_event.id
+  end
+
+  it "include BEGIN COMMIT transaction" do
+    expect(Event.all.map {|event| event.content}).to include "BEGIN COMMIT transaction"
```

Comment:
Line is too long. [87/80]<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:content` as an argument to `map` instead of a block.